src/index.html

```diff
@@ -1,17 +1,188 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname: </label>
+        <input type="text" 
```

Comment:
<img width="1363" alt="Screenshot 2023-01-22 at 14 23 13" src="https://user-images.githubusercontent.com/53228259/213915588-6a549ef2-b5a3-4a02-9e88-e4d25f3fdb7d.png">
check all tags

Reply:
> 

tags - ok
link - can you give me more precision information? what link I must to change and where, thx

Reply:
[DEMO LINK](https://alexx7356.github.io/layout_html-form/)
[TEST REPORT LINK](https://alexx7356.github.io/layout_html-form/report/html_report/)

as I see, the second link doesn't work 
but I don't understand how to resolve this problem

Reply:
now works, but still the same problem